gumbo_client/__init__.py

```diff
@@ -1,3 +1,5 @@
-__version__ = "0.2.0"
+import importlib.metadata
+
+__version__ = importlib.metadata.version('gumbo-client')
```

Comment:
@snwessel : Along with updating the version in pyproject.toml, I also updated __version__ so that it now uses that value so we only have one place to update.